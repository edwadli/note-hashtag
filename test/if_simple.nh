if true then PrintEndline "it works" else PrintEndline "you suck"
PrintEndline "this should print too"
