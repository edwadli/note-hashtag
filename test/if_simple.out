it works
this should print too
